pub/js/main.js

```diff
@@ -179,25 +183,32 @@ $(function () {
 					if (response.error_code === 0){
 						accordion.setCurrent('#step-finalize');
 					}
-					handleResponse(response, function () {}, '#step-finalize');
+					handleResponse(response, function () {}, '#step-appupgrade');
 					return response.error_code === 0
 							? $.post($('#meta-information').data('endpoint'), {command: 'upgrade:restartWebServer'})
 							: $.Deferred()
 							;
 				})
 				.then(function (response) {
-					handleResponse(response, function () {}, '#step-finalize');
+					handleResponse(response, function () {accordion.showContent('#step-finalize');}, '#step-finalize');
 					return response.error_code === 0
-							? $.post($('#meta-information').data('endpoint'), {command: 'upgrade:postUpgradeCleanup'})
+							?  accordion.setCurrent('#step-finalize')
```

Comment:
be careful with the `?` operator, priorities are often not intuitive like we think. I recommend enforcing priorities with parenthesis.